package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "magic-mirror-module-random-photo",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "Show a random photo from an url",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/skuethe/MMM-RandomPhoto.git"
+  },
+  "keywords": [
+    "MagicMirror",
+    "Module",
+    "Photo",
+    "Image",
+    "Picsum",
+    "Nextcloud"
+  ],
   "main": "MMM-RandomPhoto.js",
   "author": "skuethe",
   "license": "ISC",
   "dependencies": {
-    "jquery": "^3.1.0"
```

Comment:
Again having trouble to identify spaces via mobile app, but it seems you broke the indentation of this object. Should be 4 spaces for this childobject by respecting the 2 spaces of indentation for json files rule